deeprank/features/ResidueDensity.py

```diff
@@ -129,9 +129,13 @@ def extract_features(self):
                 atcenter = 'CA'
 
             # get the xyz of the center atom
-            xyz = self.sql.get(
-                'x,y,z', resSeq=key[1], chainID=key[0], name=atcenter)[0]
-            #xyz = np.mean(self.sql.get('x,y,z',resSeq=key[1],chainID=key[0]),0).tolist()
+            try:
+                xyz = self.sql.get(
+                    'x,y,z', resSeq=key[1], chainID=key[0], name=atcenter)[0]
+            except IndexError :
+                warnings .warn('Atom ', atcenter, ' not found for residue ', key[1], \
```

Comment:
need to remove the space between `warnings` and `.warn`

Reply:
thanks again ;)